pkg/apis/networking/validation/validation.go

```diff
@@ -93,3 +98,31 @@ func ValidateNetworkPolicyUpdate(update, old *networking.NetworkPolicy) field.Er
 	allErrs = append(allErrs, ValidateNetworkPolicySpec(&update.Spec, field.NewPath("spec"))...)
 	return allErrs
 }
+
+// ValidateIPBlock validates a cidr and the except fields of an IpBlock NetworkPolicyPeer
+func ValidateIPBlock(br *networking.IPBlock, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if len(br.CIDR) == 0 {
+		allErrs = append(allErrs, field.Required(fldPath.Child("cidr"), "ipBlock must specify a CIDR"))
```

Comment:
`field.Required(fldPath.Child("cidr"), "")` - the last error string is redundant here.